packages/suite/src/utils/suite/formatTimeLeft.tsx

```diff
@@ -0,0 +1,23 @@
+import formatDuration from 'date-fns/formatDuration';
+import intervalToDuration from 'date-fns/intervalToDuration';
+
+export const formatTimeLeft = (
+    deadline: Date,
+    locale?: Locale,
+    units: Array<keyof Duration> = ['hours', 'minutes'],
+) => {
+    const duration = intervalToDuration({
+        start: Date.now(),
+        end: deadline,
+    });
+
+    const formattedDuration: Duration = {};
+
+    units.forEach(unit => {
+        formattedDuration[unit] = duration[unit];
+    });
+
+    const formattedTimeLeft = formatDuration(formattedDuration, { locale });
```

Comment:
this works the same, but nevermind
```suggestion
    const formattedTimeLeft = formatDuration(duration, { locale, format: units });
```

Reply:
6545ed391b38ee218bd3de6f40e86656553cd05d